tests/Unit/ApparentHorizons/Test_ApparentHorizonFinder.cpp

```diff
@@ -217,6 +218,8 @@ struct MockMetavariables {
     using post_interpolation_callback =
         intrp::callbacks::FindApparentHorizon<AhA, ::Frame::Inertial>;
     using post_horizon_find_callback = PostHorizonFindCallback;
+    using horizon_find_failure_callback =
+        intrp::callbacks::ErrorOnFailedApparentHorizon;
   };
```

Comment:
add an ERROR test for a failed case (this should catch the fact you didn't update the stream operator)